clients/metadata/metadata_cassandra.go

```diff
@@ -1558,6 +1599,12 @@ func (s *CassandraMetadataService) ReadConsumerGroupByUUID(ctx thrift.Context, r
 		return nil, &shared.BadRequestError{Message: "ConsumerGroupUUID cannot be nil"}
 	}
 
+	key := request.GetConsumerGroupUUID()
```

Comment:
There are two types of keys here (dstUUID + cgName, ConsumerGroupUUID). They may conflict if cgName is empty. Maybe add some prefix to different type of keys to make sure there is no chance to conflict?

Reply:
I really hope uuids don't collide...